playbooks/roles/heketi-gluster/templates/storage-class.yml

```diff
@@ -10,3 +10,7 @@ provisioner: kubernetes.io/glusterfs
 parameters:
   resturl: "http://{{ heketi_endpoint }}"
   volumetype: {{volumetype}}
+  volumeoptions: "performance.stat-prefetch off"
```

Comment:
Please give an explanation for the options that you have changed, and please link some documentation so I can look it up.

Reply:
These options were added to make a shared instance of Minio consistent with our GlusterFS backend. Regarding the options changed, consult: https://www.mankier.com/8/mount.glusterfs and https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3/html/configuring_red_hat_openstack_with_red_hat_storage/sect-setting_up_red_hat_storage_trusted_storage_pool

Reply:
This links don't say anything about the options, just state that you can have this settings. I'd like to understand what is happening when you have this settings.

Reply:
Well the main point of these options is that multiple shared Minio instances can do consistent and safe I/O operations on a single PV provided by GlusterFS-Heketi.

Reply:
I know, but we don't just support minio. If that's the only reason these option should be parametrized.

Reply:
Otherwise, you need to give explanation why it is always better this way.

Reply:
@mcapuccini Well I think that in this case you cannot say "it can always be better". But regarding performance, the only option I touched is the following: `performance/stat-prefetch:` what it does is pre-fetching and caching file stat information when a directory is read. Speeds up operations like `ls -l`...

Reply:
Could you parametrize the options with ansible?

Reply:
Yes looks good! I think it sounds like a good option to parameterize in ansible like volumetype. I don't know what should be default. What are the costs vs benefits with "performance.stat-prefetch" on resp. off?